content/en/docs/quickstart.md

```diff
@@ -19,7 +19,7 @@ Create and run an OCI-compatible registry on your development computer using Doc
 ```console
```

Comment:
@yizha1  I think we should also claim that this quick start is using Notation v0.12.0-beta.1 at the beginning since the Notary doc has not introduced version control yet. I see Zach was confused about the testing result when using an old version of Notation.

Reply:
> @yizha1 I think we should also claim that this quick start is using Notation v0.12.0-beta.1 at the beginning since the Notary doc has not introduced version control yet. I see Zach was confused about the testing result when using an old version of Notation. 

The installation is moved to another page if you check the "Prerequisites". Maybe this was done by Zach.  I think there is a reason behind this, which is always to use the latest version. In hello-signing.md, there is version info.